### DISCUSSION TITLE
I have been pushing several commits from Lynux to GitHub (I am not using GitHub Desktop). All of a sudden after doing the exact same things I get an error message.

### DISCUSSION THREAD

**Discussion by daliaelchidiac** [community/community#121443] [New to GitHub] [answered]
<img width="599" alt="error" src="https://github.com/community/community/assets/167519731/89e50e1d-4b8f-4dbd-9ed6-58232d63429a">

**Comment by PRITHIVSAKTHIUR** [comment]
Dude, your error message indicates Git doesn't recognize 'DaliaEIChidiacCATCProject' as a valid Git repository.
Make sure about access rights in the repository to push. 
1) make sure about repo url & access right .
2) then try git push .

**Reply by daliaelchidiac** [reply, marked as answer]
Dude, as I said above I have tried all of it already. I was successfully pushing files from Lynux up until a few hours before I started receiving this error message. And I had been pushing files for days in the same repo without issues.
